.github/workflows/build_and_test.yml

```diff
@@ -200,6 +200,7 @@ jobs:
       SKIP_UNIDOC: true
       SKIP_MIMA: true
       SKIP_PACKAGING: true
+      LIVE_UI_LOCAL_STORE_DIR: "/tmp/kvStore"
```

Comment:
@zhaomin1423 All current tests have passed, and the additional verification purpose for rocksdb has been achieved. We need to remove the changes to this file before merging.

Reply:
@zhaomin1423 Please revert this change, thanks ~

Reply:
done